tgui/packages/tgui/interfaces/PreferencesMenu/QuirksPage.tsx

```diff
@@ -396,7 +397,7 @@ export function QuirksPage(props) {
 
                 <Stack.Item>
                   <StatDisplay>
-                    {positiveQuirks} / {maxPositiveQuirks}
+                    {positiveQuirks}{maxPositiveQuirks >= 0 ? ` / ${maxPositiveQuirks}` : ""}
```

Comment:
Don't show anything if there is no limit, just a plain number of count isn't useful

Reply:
Same issue as the above review. If I remove those elements then the quirks list moves upwards and out of alignment with the other half of the page.

Reply:
Fixed by commit [9bcd434](https://github.com/tgstation/tgstation/pull/81033/commits/9bcd4341d097e89ea709bd8bb10fe94cad472c31)